.github/workflows/release.yaml

```diff
@@ -3,106 +3,105 @@ name: Generate release artefact
 on:
```

Comment:
please can we remove all the unneeded indentation change and keep only real change

Reply:
Done